### PR TITLE
A new blog post "Java exceptions from container"

### PR DESCRIPTION
**Pull request by mhabrnal**
Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>

### CODE REVIEW

File: _posts/2018-05-23-java-exception-from-container.markdown [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Java exceptions from container"
+date:   2018-05-22 15:15:00 +0100
+author: Matej Habrnal <mhabrnal@redhat.com>
+visible: 1
+categories: ABRT container exception Java
+---
+As you know from a previou blog [posts], lately, we are dealing with catching
```
Comment (marusak) [inline _posts/2018-05-23-java-exception-from-container.markdown]: previous

Comment (marusak) [reply _posts/2018-05-23-java-exception-from-container.markdown]: no comma after lately

File: _posts/2018-05-23-java-exception-from-container.markdown [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+undandled exceptions from inside of containers.
```
Comment (marusak) [inline _posts/2018-05-23-java-exception-from-container.markdown]: unhandled

File: _posts/2018-05-23-java-exception-from-container.markdown [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+In this blog post, we would like to introduce a new language we are
+capable to do that for. The language is `JAVA`.
```
Comment (marusak) [inline _posts/2018-05-23-java-exception-from-container.markdown]: capable of/able to

File: _posts/2018-05-23-java-exception-from-container.markdown (hunk 1 above)
Comment (marusak) [reply _posts/2018-05-23-java-exception-from-container.markdown]: what is that [posts] ?

**Review by marusak — CHANGES_REQUESTED** [review]

**Comment by mhabrnal** [comment]
@marusak updated

**Pull request merged**